public/app/features/explore/LogsContainer.tsx

```diff
@@ -218,6 +232,9 @@ function mapStateToProps(state: StoreState, { exploreId }: { exploreId: string }
     isPaused,
     range,
     absoluteRange,
+    displayMoreLogsBtn: queries.filter(elem => !elem.hide).length === 1,
+    // currently we allow to display more logs when only one query is active
+    // due to timestamp selection issues that are going to be addressed later
```

Comment:
Used to control whether show more logs buttons are being displayed, that is only when there is one active query (as for now). Works that way due to way more complex problem with how to select a timestamp we should query things from/to, when there are multiple queries and their results set is very sparse.